### ISSUE TITLE
Bundle parameter for Symfony 4

### ISSUE THREAD

**Issue by Sluimers** [Behat/Symfony2Extension#143]
Is there a quick solution to this?

```yml
# behat.yml

default:
    suites:
        default:
           type: symfony_bundle
           bundle: App
            contexts:
                - App\Features\Bootstrap\FeatureContext:
                    doctrine: "@doctrine"
                - Imbo\BehatApiExtension\Context\ApiContext
    extensions:
        Imbo\BehatApiExtension:
            apiClient:
                base_uri: http://127.0.0.1:8000
        Behat\Symfony2Extension:
            kernel:
                bootstrap: config\behat\bootstrap.php
                class: App\Kernel
```

```
$ vendor/bin/behat features/album.feature

In SymfonySuiteGenerator.php line 58:
                                                  
  The bundle "App" does not exist in the project  
                                                  
In Kernel.php line 218:
                                                                                                                                             
  Bundle "App" does not exist or it is not enabled. Maybe you forgot to add it in the registerBundles() method of your App\Kernel.php file?                                                                                                     
```

**Comment by sroze** [comment]
I've just merged #141, it should help. Can you try `master` and let me know?

**Comment by Sluimers** [comment]
It works.
